source/js/buyers-guide/search.js

```diff
@@ -0,0 +1,107 @@
+const ALL_PRODUCTS = document.querySelectorAll(`figure.product-box`);
+
+const filters = [
```

Comment:
let's cap this as well
```suggestion
const FILTERS = [
```